src/revlanguage/functions/math_helper/Func_SmoothenTimeline.h

```diff
@@ -0,0 +1,48 @@
+#ifndef Func_SmoothenTimeline_H
+#define Func_SmoothenTimeline_H
```

Comment:
This should probably be SmoothTimeLine.  The verb for making something smooth is "to smooth".  Apparently some dictionaries list "smoothen" as well, but it sounds a bit odd.

Reply:
Done.